app/assets/stylesheets/users/profile/profile.scss

```diff
@@ -0,0 +1,109 @@
+.user-profile {
+  padding-left: 5%;
+  padding-right: 5%;
+}
+
+.profile-sections {
+  display: grid;
+  grid-gap: 70px;
+  grid-template-columns: repeat(10, 1fr);
+  grid-template-rows: 150px auto auto; 
+  padding-top: 100px; //as place holder. will not be needed when profile header is put in place. 
+  
+  [data-title="badges"] {
+    grid-column: 1 / 4;
+    grid-row: 1 / 2;
+  }
+  [data-title="donations-by-causes"] {
+    grid-column: 4 / 8;
+    grid-row: 1 / 2;
+  }
+  [data-title="past-donations"] {
+    grid-column: 8 / 11;
+    grid-row: 1 / 4;
+  }
+  [data-title="philosophy"] {
+    grid-column: 1 / 4;
+    grid-row: 2 / 3;
+  }
+  [data-title="favorite-organizations"] {
+    grid-column: 4 / 8;
+    grid-row: 2 / 4;
+  }
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////
+// Still experimenting with these responsive designs. Will do more exploraion later. 
+  @media all and (max-width: 900px){
+    display: grid;
+    grid-gap: 50px;
+    grid-template-columns: repeat(7, 1fr);
+    grid-template-rows: 150px auto auto; 
+    
+    [data-title="badges"] {
+      grid-column: 1 / 4;
+      grid-row: 1 / 2;
+    }
+    [data-title="donations-by-causes"] {
+      grid-column: 4 / 8;
+      grid-row: 1 / 2;
+    }
+    [data-title="philosophy"] {
+      grid-column: 1 / 4;
+      grid-row: 2 / 3;
+    }
+    [data-title="favorite-organizations"] {
+      grid-column: 4 / 8;
+      grid-row: 2 / 4;
+    }
+    [data-title="past-donations"] {
+      grid-column: 1 / 4;
+      grid-row: 3 / 4;
+    }
+  }
+
+  @media all and (max-width: 650px){
+    display: flex;
+    flex-direction: column;
+    
+    [data-title="badges"] { 
+      order: 0; 
+      margin: 5px;
+    }
+    [data-title="philosophy"] { 
+      order: 1;
+      margin: 5px; 
+    }
+    [data-title="donations-by-causes"] { 
+      order: 2; 
+      margin: 5px;
+    }
+    [data-title="past-donations"] { 
+      order: 3; 
+      margin: 5px;
+    }
+    [data-title="favorite-organizations"] { 
+      order: 4; 
+      margin: 5px;
+    }
+  }
+//////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
the comment here isn't necessary

Reply:
it's just as a visual cue when scrolling.

Reply:
If you want visual cues, you can space out the selectors, but we shouldn't be adding comments for layout purposes